app/Domain/Goalcanvas/Controllers/EditCanvasItem.php

```diff
@@ -201,8 +201,8 @@ public function post($params): Response
                             'parent' => $params['parent'] ?? null,
                             "id" => $params['itemId'],
                             'kpi' => $params['kpi'] ?? '',
-                            'startDate' => format(value: $params['startDate'], fromFormat: FromFormat::UserDateStartOfDay)->isoDateTime(),
-                            'endDate' => format(value: $params['endDate'], fromFormat: FromFormat::UserDateEndOfDay)->isoDateTime(),
+                            'startDate' => !empty($params['startDate'])?format(value: $params['startDate'], fromFormat: FromFormat::UserDateStartOfDay)->isoDateTime():'',
```

Comment:
Few items missing from this file:
- No repo calls (they should all be service calls. If there is no service, just create a wrapper service method
- CANVAS_NAME is still being used